src/Query/WhereGroup.php

```diff
@@ -15,8 +15,8 @@ class WhereGroup implements Iterator, Countable
     /**
      * Conjunction values
      */
-    const AND = 0;
-    const OR = 1;
+    const int AND = 0;
+    const int OR = 1;
```

Comment:
I think this would be better as an enum. The value doesn't actually mean anything useful.